code/style.css

```diff
@@ -1,7 +1,385 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-gap: 5px;
+  margin: 0px;
+  width: 100%;
+  font-family: 'Montserrat', sans-serif;
+  background-color: rgb(247, 217, 216);
+  color: rgb(238, 208, 209);
+}
+
+header {
+  grid-column: span 6;
+}
+
+.header {
+  background-image: url("https://images.unsplash.com/photo-1445140463371-d8036feedc2f?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8aGVhZGVyJTIwaW1hZ2UlMjBiZWVyfGVufDB8fDB8fA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=400&q=60");
+  height: 400px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+h1 {
+  margin: 0px;
+  text-align: center;
+  padding-top: 130px;
+  font-size: 60px;
+  font-weight: 200;
+}
+
+h2 {
+  text-align: center;
+  margin: 0px;
+  font-size: 27px;
+  font-weight: 200;
+  padding: 0 3px;
+}
+
+.navigator-top {
+  grid-column: span 6;
+  display: grid;
+  grid-template-columns: 1fr 1fr auto 1fr 1fr;
```

Comment:
If you want the space between to be equal everywhere, one could write the code like this:
`grid-template-columns: 1fr 1fr auto 1fr 1fr;`